man/pdp.rank.Rd

```diff
@@ -0,0 +1,44 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/plotter.R
+\name{pdp.rank}
+\alias{pdp.rank}
+\title{Given an interpreter object with choice of PDP ranking methodology
+       (default: 'Variance'), produce PDP 'ranking' scores by feature. Optionally,
+       permits a new observation to weight the PDP function and rankings.}
+\usage{
+pdp.rank(
+  object,
+  rank.method = "Variance",
+  new.obs = NULL,
+  feature = NULL,
+  qt = 5
+)
+}
+\arguments{
+\item{object}{The Interpreter class that we want understand the PDP ranking scores of.}
+
+\item{rank.method}{A string to select which PDP ranking methodology. Should be one of
+c("Variance", "FO.Derivative"). When set to "Variance" the PDP functions are ranked by variance
+of the function over the range of grid.points and when set to "FO.Derivative" the PDP functions
+are ranked by empirical first order derivative.}
+
+\item{new.obs}{A data frame with equivalent name/features of the data that trained the
```

Comment:
This description should also be more descriptive. The predictor is an object rather than a parameter. The new.obs is really a single observation that takes the form of a data frame with a single row. We should also say exactly how this observation influences the weighting of the PDP: "The standard PDP function is a simple average of the ICE functions, this option produces a personalized PDP curve that is the weighted average of the ICE functions where the kth ICE function is given the weight the random forest gives observation k when predicting for new.obs"